Description: Use libpq-dev instead libpq-fe.h from upstream.
Forwarded: yes
Author: Julián Moreno Patiño <darkjunix@gmail.com>
Last-Update: 2011-06-18
--- a/hydra-postgres.c
+++ b/hydra-postgres.c
@@ -14,7 +14,7 @@ void dummy_postgres() {
 }
 #else
 
-#include "libpq-fe.h"           // Postgres connection functions
+#include <postgresql/libpq-fe.h>           // Postgres connection functions
 #include <stdio.h>
 
 #define DEFAULT_DB "template1"
