Description: Only using the supported protocols
Forwarded: no
Author: Julián Moreno Patiño <darkjunix@gmail.com>
Last-Update: 2013-07-15
--- a/hydra-gtk/src/interface.c
+++ b/hydra-gtk/src/interface.c
@@ -226,12 +226,13 @@
   gtk_widget_set_name(cmbProtocol, "cmbProtocol");
   gtk_widget_show(cmbProtocol);
   gtk_table_attach(GTK_TABLE(table8), cmbProtocol, 1, 2, 4, 5, (GtkAttachOptions) (GTK_EXPAND | GTK_SHRINK), (GtkAttachOptions) (GTK_EXPAND), 0, 0);
-  cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "afp");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "asterisk");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "cisco");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "cisco-enable");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "cvs");
+  #ifndef __GNU__
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "firebird");
+  #endif
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "ftp");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "ftps");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "http-head");
@@ -253,9 +254,7 @@
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "ldap3-digestmd5");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "mssql");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "mysql");
-  cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "ncp");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "nntp");
-  cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "oracle");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "oracle-listener");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "oracle-sid");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "pcnfs");
@@ -267,7 +266,6 @@
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "rlogin");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "rsh");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "s7-300");
-  cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "sapr3");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "sip");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "smb");
   cmbProtocol_items = g_list_append(cmbProtocol_items, (gpointer) "smtp");
