Description: Remove troubled files to avoid FTBFS.
Forwarded: no
Author: Julián Moreno Patiño <darkjunix@gmail.com>
Last-Update: 2012-02-16
--- a/Makefile
+++ /dev/null
@@ -1,5 +0,0 @@
-all:
-	@echo Error: you must run "./configure" first
-
-clean:
-	cp -f Makefile.orig Makefile
--- a/Makefile.am
+++ b/Makefile.am
@@ -69,5 +69,4 @@ install:	strip
 
 clean:
 	rm -rf xhydra pw-inspector hydra *.o core *.core *.stackdump *~ Makefile.in Makefile dev_rfc hydra.restore arm/*.ipk arm/ipkg/usr/bin/* hydra-gtk/src/*.o hydra-gtk/src/xhydra hydra-gtk/stamp-h hydra-gtk/config.status hydra-gtk/errors hydra-gtk/config.log hydra-gtk/src/.deps hydra-gtk/src/Makefile hydra-gtk/Makefile
-	cp -f Makefile.orig Makefile
 
